### PR TITLE
Oc 11772:Casebook PDF unabled to include attached images

### PR DESCRIPTION
**Pull request by Taoli2018**

### CODE REVIEW

File: src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+package core.org.akaza.openclinica.service.crfdata.xform;
+
+import org.springframework.util.LinkedMultiValueMap;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class EnketoPDFRequest extends LinkedMultiValueMap {
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java]: let's remove the extension and retry

File: src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	public String getLandscape() {
+		return landscape;
+	}
+
+	public void setLandscape(String landscape) {
+		this.landscape = landscape;
+		super.add("landscape", landscape);
+	}
+
+	public InstanceAttachment getInstanceAttachments() {
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java]: add @JsonProperty annotation on getter and setter methods too

File: src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java (hunk 1 above)
Comment (Taoli2018) [reply src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java]: done

File: src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java (hunk 2 above)
Comment (Taoli2018) [reply src/main/java/core/org/akaza/openclinica/service/crfdata/xform/EnketoPDFRequest.java]: done

**Comment by svadla-oc** [comment]
👍

**Pull request merged**